tests/test_apps/app_with_callbacks/custom_callback/my_callbacks_impl.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
+
+import logging
+
+from omegaconf import DictConfig
+
+from hydra._internal.callback import Callback
+from hydra.core.utils import JobReturn
+
+log = logging.getLogger(__name__)
+
+
+class CustomCallback(Callback):
```

Comment:
For simplicity, can you move the callback implementation to tests/test_apps/app_with_callbacks/custom_callback/my_app.py?